sdk/core/Azure.Core/src/DynamicData/DynamicData.cs

```diff
@@ -197,11 +197,13 @@ private object ConvertType(object value)
             try
             {
 #if NET6_0_OR_GREATER
+
```

Comment:
nit: if there's no change other than whitespace, it would be best to revert the file.  This is helpful for people who come back to the PR later to be able to see what files have meaningful changes in the list of files changed by the PR.